TLC59116.h

```diff
@@ -446,7 +446,7 @@ class TLC59116 : public TLC59116_Unmanaged {
   private:
     // FIXME: move Power_Up_Register_Values to FLASH
     /// The state of the device on power-up/reset.
-    static const unsigned char Power_Up_Register_Values[TLC59116_Unmanaged::Control_Register_Max + 1] PROGMEM;
+    static const unsigned char Power_Up_Register_Values[TLC59116_Unmanaged::Control_Register_Max + 1];
```

Comment:
I intended to keep those in progmem, to use less ram. Did I do it wrong?

Reply:
This may have come from a different version.  I just verified it compiles either way -- so disregard and keep PROGMEM